LICENSE

```diff
@@ -1,2 +1,202 @@
-Unless specifically indicated otherwise in a file, files are licensed
-under the Apache 2.0 license, as can be found in: apache-2.0.txt
+
+                                 Apache License
```

Comment:
Technically the license statement is correct: if you obtain the source code directly from the Git repository, it's covered by the Apache license, and we do not need this file to mention the fact that we also make releases available as tarballs with a GPL license. However it would be less confusing if the `LICENSE` file in the source code said that this source is available under either license at the recipient's choice. Doing this also requires changing the license statements in individual files, so it doesn't belong in the same pull request, but it belongs in a follow-up which would have to add a reference to the GPL here. It would be simpler to keep the Apache license in `apache-2.0.txt`, and in the follow-up add a file `GPL-2.txt` and add a reference to it in `LICENSE`.

Reply:
Yes, I agree with all of that but we can't do it until we've changed the release process to not distribute 2 different tarballs with different license boilerplate text in the source files.